src/pages/permissions/PermissionIdentities.tsx

```diff
@@ -294,7 +300,7 @@ const PermissionIdentities: FC = () => {
                 selectedNames={selectedIdentityIds}
                 setSelectedNames={setSelectedIdentityIds}
                 processingNames={[]}
-                filteredNames={nonTlsUsers.map((identity) => identity.id)}
+                filteredNames={identities.map((identity) => identity.id)}
```

Comment:
related to prior comments, this should reflect fine grained identities and not all of them

Reply:
As above.

Reply:
nothing changed here.